crates/swc_css_minifier/tests/fixture/compress-calc/discard-zero/output.min.css

```diff
@@ -1 +1 @@
-.class1{width:calc(100vw/2 - 6px)}.class2{width:500px}.class3{width:calc(0 - 10px)}.class4{width:calc(0 - 1px - 1em)}.class5{width:calc(0 - (100vw - 10px)/2)}.class6{width:calc(10px - 100vw)}.class7{width:calc(0px - (100vw - 10px)*2)}.class8{width:calc(-10px - 100vw)}.class9{width:calc(0px - var(--foo,4px)/2)}
+.class1{width:calc(-6px + 50vw)}.class2{width:500px}.class3{width:calc(0 - 10px)}.class4{width:calc(0 - 1em - 1px)}.class5{width:calc(0 + 5px - 50vw)}.class6{width:calc(10px - 100vw)}.class7{width:calc(20px - 200vw)}.class8{width:calc(-10px - 100vw)}.class9{width:calc(0px - .5*var(--foo,4px))}
```

Comment:
`.class9{width:calc(0px - .5*var(--foo,4px))}` is valid compression, but `.class9{width:calc(0px - var(--foo,4px)/2)}` has less characters, can we avoid it in cases when characters are more (yes, it is not by spec, but in compresion it is normal to skip some cases in favor size)

Reply:
Or we can put TODOs for future improvements

Reply:
I put a TODO on the `simplify_calc_operator_node_invert` function. I think it's a bit tricky to manage all cases.